packages/contracts-bedrock/contracts/L1/L1CrossDomainMessenger.sol

```diff
@@ -13,34 +13,26 @@ import { CrossDomainMessenger } from "../universal/CrossDomainMessenger.sol";
  *         interface instead of interacting with lower-level contracts directly.
  */
 contract L1CrossDomainMessenger is CrossDomainMessenger {
-    /**
-     * @notice Contract version number.
-     */
-    uint8 public constant VERSION = 1;
```

Comment:
Removing `VERSION` in favor of using a base contract like the `Semver` we have in `contracts-periphery` instead.

Reply:
we could import that contract here?